int32 numcluster
float64[30] baselink_x
float64[30] baselink_y
float64[30] baselink_z
float64[30] twodim_x
float64[30] twodim_y
float64[30] twodim_z


